NewHorizons/Builder/Props/DetailBuilder.cs

```diff
@@ -345,8 +347,9 @@ private static void FixComponent(Component component, GameObject planetGO, bool
 
             else if (component is NomaiInterfaceOrb orb)
             {
-                orb._parentAstroObject = planetGO.GetComponent<AstroObject>();
-                orb._parentBody = planetGO.GetComponent<OWRigidbody>();
+                // detect planet gravity
+                var gravityVolume = planetGO.GetAttachedOWRigidbody().GetAttachedGravityVolume();
+                orb.GetComponent<ConstantForceDetector>()._detectableFields = gravityVolume ? new ForceVolume[] { gravityVolume } : new ForceVolume[] { };
```

Comment:
ConstantForceDetector should always exist on the orbs, as NomaiInterfaceOrb also uses it